target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -335,11 +335,19 @@ wndr3700v5)
 wt3020)
 	ucidef_set_led_default "power" "power" "$board:blue:power" "0"
 	;;
-y1|\
+y1)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
```

Comment:
This shouldn't be required. The LED is already handled by target/linux/ramips/base-files/etc/diag.sh.